src/migrations/0072-release-semver-prerelease-build-metadata.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "semver prerelease" VARCHAR(255) NOT NULL DEFAULT '',
```

Comment:
When we added the rest semver fields that had 0 as the default value we didn't notice any performance issue.
I realize that NVARCHAR is might not have the same const as an INTEGER field, but on the other hand PG11+ claims that it will only add the default values lazily, so that the ALTER TABLE doesn't cause a full table lock.
See: https://www.postgresql.org/docs/current/ddl-alter.html#DDL-ALTER-ADDING-A-COLUMN

Reply:
Thanks for the reference. Should be fine then 👍